apps/autofix/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  autofix-container:
+    build:
+      context: ../..
+      dockerfile: apps/autofix/src/container-server/Dockerfile
+      args:
+        NODE_EXTRA_CA_CERTS: ${NODE_EXTRA_CA_CERTS:-}
```

Comment:
This probably should be a constant of `/etc/ssl/certs/Cloudflare_CA.pem` https://github.com/jahands/autofix-agent/pull/29/files#r2105223410